README.md

```diff
@@ -16,6 +16,12 @@ cd psammead && npm install
 
 NB, this automatically runs `npm run install:packages` in a postinstall step for you.
 
+### Run tests
```

Comment:
Not sure if this is in scope of this issue, but we should write about how testing is run at a repo level, and its is not the responsibility of the package to setup testing